backend/src/index.ts

```diff
@@ -21,6 +24,9 @@ app.use(bodyParser.urlencoded({ limit: "20mb", extended: true }));
 
 app.use("/api/users", users);
 app.use("/api/auth", auth);
+app.use("/api/tenants", tenants);
+app.use("/api/owners", owners);
+app.use("/api/realEstates", realEstates);
```

Comment:
Arriba metete un 
```
import authMiddleware from './middlewares/auth';
```
Y acá entonces le pasas como segundo param ese middleware y voilá

```suggestion
app.use("/api/tenants", authMiddleware, tenants);
app.use("/api/owners", authMiddleware, owners);
app.use("/api/realEstates", authMiddleware, realEstates);
```

Reply:
o 'auth' solo no?. Digo, para que quede igual que el otro, si no le agrego la palabra middleware al otro tmb